alibi/explainers/anchors/anchor_image.py

```diff
@@ -467,19 +467,44 @@ def explain(self,  # type: ignore[override]
         image
             Image to be explained.
         p_sample
-            Probability for a pixel to be represented by the average value of its superpixel.
+            Probability for a pixel to be represented by the average value of its superpixel. The missingness of a
+            superpixel (i.e. querying the model on a reduced input) is simulated by randomly turning it on and off
+            with a probability `p_sample`.
```

Comment:
I think if `images_background` we replace with the other image, not the average.